README.md

```diff
@@ -1,135 +1,168 @@
-2. **Run the CLI:** Execute the following command in your terminal:
-
-   ```bash
-   npx https://github.com/google-gemini/gemini-cli
-   ```
-
-   Or install it with:
-
-   ```bash
-   npm install -g @google/gemini-cli
-   ```
-
-   Then, run the CLI from anywhere:
-
-   ```bash
-   gemini
-   ```
-
-### With Homebrew
-
-1. **Prerequisites:** Ensure you have [Homebrew](https://brew.sh/) installed.
-2. **Install the CLI:** Execute the following command in your terminal:
-
-   ```bash
-   brew install gemini-cli
-   ```
-
-   Then, run the CLI from anywhere:
+```bash
+# Using npx (no installation required)
+npx https://github.com/google-gemini/gemini-cli
+```
 
-   ```bash
-   gemini
-   ```
+#### Install globally with npm
+```bash
+npm install -g @google/gemini-cli
+```
 
-### Common Configuration steps
+#### Install globally with Homebrew (macOS/Linux)
+```bash
+brew install gemini-cli
+```
 
-3. **Pick a color theme**
-4. **Authenticate:** When prompted, sign in with your personal Google account. This will grant you up to 60 model requests per minute and 1,000 model requests per day using Gemini.
+#### System Requirements
+- Node.js version 20 or higher
+- macOS, Linux, or Windows
+
+## 📋 Key Features
+
+With Gemini CLI you can:
+
+- **Code Understanding & Generation**
+  - Query and edit large codebases
+  - Generate new apps from PDFs, images, or sketches using multimodal capabilities
+  - Debug issues and troubleshoot with natural language
+  
+- **Automation & Integration**
+  - Automate operational tasks like querying pull requests or handling complex rebases
+  - Use MCP servers to connect new capabilities, including [media generation with Imagen, Veo or Lyria](https://github.com/GoogleCloudPlatform/vertex-ai-creative-studio/tree/main/experiments/mcp-genmedia)
+  - Run non-interactively in scripts for workflow automation
+  
+- **Advanced Capabilities**
+  - Ground your queries with built-in [Google Search](https://ai.google.dev/gemini-api/docs/grounding) for real-time information
+  - Conversation checkpointing to save and resume complex sessions
+  - Custom context files (GEMINI.md) to tailor behavior for your projects
+
+- **🔗 GitHub Integration**
+  - Use the Gemini CLI GitHub Action for automated PR reviews
+  - Automated issue triage and on-demand AI assistance directly in your repositories
+  - Seamless integration with your GitHub workflows
+
+## 🔐 Authentication Options
+
+Choose the authentication method that best fits your needs:
+
+### Option 1: OAuth login (Using your Google Account)
+**✨ Best for:** Individual developers as well as anyone who has a Gemini Code Assist License. (see [quota limits and terms of service](https://cloud.google.com/gemini/docs/quotas) for details)
+
+**Benefits:**
+- **Free tier**: 60 requests/min and 1,000 requests/day
+- **Gemini 2.5 Pro** with 1M token context window
+- **No API key management** - just sign in with your Google account
```

Comment:
We would want to make this consistent - one line item has a ":", one line item has nothing, and another line item has a " - "